core/ledger/kvledger/txmgmt/pvtstatepurgemgmt/purge_mgr_test.go

```diff
@@ -28,6 +28,16 @@ var testEnvs = []privacyenabledstate.TestEnv{
 	&privacyenabledstate.CouchDBCommonStorageTestEnv{},
 }
 
+const (
+	levelDBtestEnvName = "levelDB_TestEnv"
+	couchDBtestEnvName = "couchDB_TestEnv"
+)
+
+var testEnvsMap = map[string]privacyenabledstate.TestEnv{
```

Comment:
Why not just replace testEnvs with the map:

```
var testEnvs = map[string]privacyenabledstate.TestEnv{
	"levelDB": &privacyenabledstate.LevelDBCommonStorageTestEnv{},
	"couchDB": &privacyenabledstate.CouchDBCommonStorageTestEnv{},
}
```

and then you can iterate over it with:

```
for _, dbEnv := range testEnvs {
	t.Run(dbEnv.GetName(), func(t *testing.T) { testPurgeMgrForCommittingPvtDataOfOldBlocks(t, dbEnv) })
}
```
and reference directly with `coudbDB` and `levelDB` adding the constant and the secondary map is just an abstraction